go.mod

```diff
@@ -25,7 +27,7 @@ require (
 	github.com/spf13/viper v1.15.0
 	golang.org/x/crypto v0.21.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/mount-utils v0.23.0
+	k8s.io/mount-utils v0.30.1
```

Comment:
This is the actual bump, all other changes in go.mod and in vendor as caused the indirect dependencies and the mount-utils code itself.

Reply:
golang vendoring... 😅